src/FileService.Business/Commands/File/CreateFilesCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using LT.DigitalOffice.FileService.Broker.Publishes.Interfaces;
+using LT.DigitalOffice.FileService.Business.Commands.File.Interfaces;
+using LT.DigitalOffice.FileService.Data.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.Kernel.BrokerSupport.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Models.Project;
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.ProjectService.Broker.Requests.Interfaces;
+using Microsoft.AspNetCore.Http;
+
+namespace LT.DigitalOffice.FileService.Business.Commands.File
+{
+  public class CreateFilesCommand : ICreateFileCommand
+  {
+    private readonly IFileRepository _fileRepository;
+    private readonly IResponseCreator _responseCreator;
+    private readonly IDbFileMapper _mapper;
+    private readonly IAccessValidator _accessValidator;
+    private readonly IProjectService _projectService;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IPublish _publish;
+
+    public CreateFilesCommand(
+      IResponseCreator responseCreator,
+      IFileRepository fileRepository,
+      IDbFileMapper mapper,
+      IAccessValidator accessValidator,
+      IProjectService projectService,
+      IHttpContextAccessor httpContextAccessor,
+      IPublish publish)
+    {
+      _fileRepository = fileRepository;
+      _responseCreator = responseCreator;
+      _mapper = mapper;
+      _accessValidator = accessValidator;
+      _projectService = projectService;
+      _httpContextAccessor = httpContextAccessor;
+      _publish = publish;
+    }
+
+    public async Task<OperationResultResponse<List<Guid>>> ExecuteAsync(Guid entityId, FileAccessType access, IFormFileCollection uploadedFiles)
+    {
+      List<ProjectUserData> users = await _projectService.GetProjectUsersAsync(new List<Guid>() { _httpContextAccessor.HttpContext.GetUserId() }, new());

```

Comment:
you do not use List errors
In this place no need create the list

Reply:
done